packages/eslint-plugin/src/rules/naming-convention.ts

```diff
@@ -519,27 +523,48 @@ export default createRule<Options, MessageIds>({
       // #region accessor
 
       'Property[computed = false]:matches([kind = "get"], [kind = "set"])': {
-        validator: validators.accessor,
+        validator: validators.classicAccessor,
         handler: (node: TSESTree.PropertyNonComputedName, validator): void => {
           const modifiers = new Set<Modifiers>([Modifiers.public]);
           handleMember(validator, node, modifiers);
         },
       },
 
-      'MethodDefinition[computed = false]:matches([kind = "get"], [kind = "set"])':
-        {
-          validator: validators.accessor,
-          handler: (
-            node: TSESTree.MethodDefinitionNonComputedName,
-            validator,
-          ): void => {
-            const modifiers = getMemberModifiers(node);
-            handleMember(validator, node, modifiers);
-          },
+      [[
+        'MethodDefinition[computed = false]:matches([kind = "get"], [kind = "set"])',
+        ':matches(TSAbstractMethodDefinition)[computed = false][kind="get"]',
+        ':matches(TSAbstractMethodDefinition)[computed = false][kind="set"]',
```

Comment:
```suggestion
        'TSAbstractMethodDefinition[computed = false]:matches([kind="get"], [kind="set"])'
```

- `:matches` accepts a list of things and asserts the node matches at least one of them. So `:matches(T)` is the same as just `T`.
- We can use `:matches` to join these two lines, as in the previous line.